.travis.yml

```diff
@@ -20,7 +26,7 @@ r_github_packages:
   - USGS-R/rloadest@v0.4.2
 
 after_success:
-  - Rscript -e 'covr::codecov(); covr::coveralls()'
+  - Rscript -e 'library(covr);covr::codecov(); covr::coveralls()'
```

Comment:
No I don't think it is needed, I was thinking about taking that back out. Let's go ahead and do that.